GERTi/GERTiClient.lua

```diff
@@ -94,9 +94,10 @@ end
 local handler = {}
 handler.CloseConnection = function(_, _, port, connectDex)
 	if connections[connectDex]["nextHop"] then
-		transInfo(connections[connectDex]["nextHop"], connections[connectDex]["sendM"], connections[connectDex]["port"], "CloseConnection", connectDex)
+		transInfo(connections[connectDex]["nextHop"], connections[connectDex]["sendM"], connections[connectDex]["port"], "CloseConnection", tostring(connectDex))
 	else
 		computer.pushSignal("GERTConnectionClose", connections[connectDex]["origin"], connections[connectDex]["dest"], connections[connectDex]["ID"])
+		cPend[connections[connectDex]["dest"]..connections[connectDex]["origin"]..connections[connectDex]["ID"]]=nil
```

Comment:
Extraneous, please see main review

Reply:
I can confirm through testing that cPend is not cleared when this closes. cPend is filled twice per computer once both computers have established a socket, once is cPend[dest..origin..ID] and the other is cPend[origin..dest..ID]. Only one of them is cleared, at the end of RouteOpen, but the other one is never cleared, causing the computer to be unable to respond to further OpenRoute events.